app/views/content/international-returners.md

```diff
@@ -21,12 +21,12 @@ accordion:
       partial: content/international-returners/check-your-qualifications
     Find your next role:
       partial: content/international-returners/find-your-next-role
-    Support for shortage subjects:
-      partial: content/international-returners/support-for-shortage-subjects
+    Support for teachers of shortage subjects:
```

Comment:
Given that this is a step by step guide and all the other steps are actions, should this one be as well for consistency? Could this step then be 'talk to an adviser'?

Reply:
You can only get an adviser if you teach specific subjects so probs not :)

Reply:
@laurahermionetennant92 true, but we do still do this on other pages ([like the returners page](https://getintoteaching.education.gov.uk/returning-to-teaching/#get-a-return-to-teaching-adviser) and then just give details about eligibility underneath. Just might make the heading easier to read/more consistent with the other steps.